beacon-chain/core/electra/consolidations.go

```diff
@@ -258,6 +265,18 @@ func ProcessConsolidationRequests(ctx context.Context, st state.BeaconState, req
 			continue
 		}
 
+		if curEpoch < srcV.ActivationEpoch+params.BeaconConfig().ShardCommitteePeriod {
```

Comment:
Overflow shouldn't be possible here since we verified that srcV is active, but we could defensively future proof here with an overflow check or math.SafeAdd. 

What do you think?

Reply:
599805f6a727fca9a4d9bcdf314bd6dd19e09be1